src/shared/components/navigation/navigation.component.js

```diff
@@ -0,0 +1,34 @@
+import React, { memo } from 'react';
+
+import { Container, List, ListItem, ItemLink } from './navigation.styles';
+
+const Navigation = () => {
+  return (
+    <Container>
+      <List>
+        <ListItem>
+          <ItemLink exact to="/">
```

Comment:
Generalnie pary (link, copy) możesz sobie wrzucić do jakiejść stałej i po prsotu użyć tutaj `.map`

Reply:
Tak wiem, poprawię to w momencie jak przemyślę i stworzę strukturę projektu